float64 speed
int32 emergencyflag
float64 curvature
float64 Acceltarget
int32 genAccelMode
float64 distSafe
float64 distxobj
float64 distyobj
float64 egospeed
float64 egospeedinit
float64 accelCar
float64 THW_FC
float64 accelFollowingFC
float64 relspeedFC
float64 objspeedFC
float64 distPercentage
float64 followingTime
float64 accelfollowingAC
float64 accelfollowingMaxAC
float64 accelfollowingMinAC
float64 safeTimeInterval
float64 timeCollisionEgo
float64 timeCollisionObj
float64 Distw_obs_min
float64 Dsafe
float64 testVar
float64 testVar1
